src/pyscipopt/conshdlr.pxi

```diff
@@ -452,3 +460,19 @@ cdef SCIP_RETCODE PyConsGetdivebdchgs (SCIP* scip, SCIP_CONSHDLR* conshdlr, SCIP
     success[0] = False
     infeasible[0] = False
     return SCIP_OKAY
+
+cdef SCIP_RETCODE PyConsGetPermSymGraph (SCIP* scip, SCIP_CONSHDLR* conshdlr, SCIP_CONS* cons, SYM_GRAPH* graph,
+                                         SCIP_Bool* success) noexcept with gil:
+    # TODO
+    PyConshdlr = getPyConshdlr(conshdlr)
+    PyConshdlr.consgetpermsymgraph()
+    success[0] = False
+    return SCIP_OKAY
+
+cdef SCIP_RETCODE PyConsGetSignedPermSymGraph (SCIP* scip, SCIP_CONSHDLR* conshdlr, SCIP_CONS* cons, SYM_GRAPH* graph,
+                                               SCIP_Bool* success) noexcept with gil:
+    # TODO
```

Comment:
@Opt-Mucca Could you please elaborate more here? so that in the future people would know what needs to be done

Reply:
I add issue #799 to track this so that we could merge this and deal with this later.